ui/Makefile

```diff
@@ -143,3 +143,8 @@ $(GOBINDATA_DEBUG_TARGET): $(GOBINDATA_DEBUG_DEPS) bower.installed debug/$(INDEX
 
 watch: debug
 	gulp watch
+
+.PHONY:
+clean:
+	rm -f npm.installed bower.installed tsd.installed $(INDEX)
```

Comment:
I prefer the other form; I think it's generally better for `make clean` to make an effort to restrict its actions to things that may have been put there by the `Makefile` instead of relying on external metadata and deleting anything unrecognized.